influxdb/README.md

```diff
@@ -0,0 +1,72 @@
+# Metric Source: InfluxDB
+
+To import a metric from InfluxDB, ts-bridge regularly runs a configured query
+against the [InfluxDB API](https://docs.influxdata.com/influxdb/v1.7/tools/api/).
+
+Metrics imported from InfluxDB are defined in the `influxdb_metrics` section of
+`app/metrics.yaml`. The following parameters need to be specified for each
+metric:
+
+*   `name`: base name of the metric. While exporting to Stackdriver, this name
+    will be prefixed with `custom.googleapis.com/influxdb/`.
+*   `query`: [InfluxQL](https://docs.influxdata.com/influxdb/v1.7/query_language/)
+    query. This needs to return a single time series (tags/labels are not
+    supported yet).
+*   `database`: InfluxDB database which the query is to be executed against.
+*   `endpoint`: base URL of your InfluxDB service.
+*   `auth.username`: username used for authentication. If none is provided,
+    no authentication will be passed.
+*   `auth.password`: password used for authentication.
+*   `time_aggregated`: a boolean flag describing whether the query is
+    time aggregated (i.e. containing `GROUP BY time(x)`).
+*   `cumulative`: a boolean flag describing whether query result should be
+    imported as a cumulative metric (a monotonically increasing counter). See
+    [Cumulative metrics](#cumulative-metrics) section below for more details.
+*   `destination`: name of the Stackdriver destination that query result will be
+    written to. Destinations need to be explicitly listed in the
+    `stackdriver_destinations` section of the configuration file.
+
+All parameters other than `auth` and the boolean flags (`time_aggregated` and
+`cumulative` defaults to `false`) are required.
+
+For metrics that have measurements more often than every minute, you might
+consider time aggregating the query to avoid importing too many points. For
+example, a gauge metric query such as:
+
+`SELECT free_disk_space FROM sys_disk`
+
+can be time aggregated as the following:
+
+`SELECT MEAN(free_disk_space) FROM sys_disk GROUP BY time(1m)`.
```

Comment:
Perhaps mention again here that this also requires `time_aggregated` to be set to true?

Reply:
Good point, added! :)